### ISSUE TITLE
@pothos/plugin-prisma generated types doesn't import from prisma

### ISSUE THREAD

**Issue by JipSterk** [hayes/pothos#286]
I am working on a project with federated microservices and wanted to move my code over to pothos because i saw that you are working on a federation plugin for pothos.

in this project i make use of lerna and yarn workspaces. 
my root package.json i made added prisma to the `nohoist` option to avoid hoisting.

```json
  "workspaces": {
    "packages": [
      "packages/*"
    ],
    "nohoist": [
      "**/.prisma/**",
      "**/.prisma",
      "**/@prisma/**",
      "**/@prisma",
      "**/prisma/**",
      "**/prisma",
    ]
  },
```

i've added the photos generator like so:

```prisma
generator pothos {
    provider = "prisma-pothos-types"
}
```

but the generated types don't seem to import the types from prisma even though it knows were its located

```ts
import type { , ,  } from "/Users/***/Repos/***/packages/accounts-service/node_modules/@prisma/client";
export default interface PrismaTypes {
    User: {
        Shape: User;
        Include: Prisma.UserInclude;
        Where: Prisma.UserWhereUniqueInput;
        Fields: "restaurateurApplications";
        ListRelations: "restaurateurApplications";
        Relations: {
            restaurateurApplications: {
                Shape: RestaurateurApplication[];
                Types: PrismaTypes["RestaurateurApplication"];
            };
        };
    };
    RestaurateurApplication: {
        Shape: RestaurateurApplication;
        Include: Prisma.RestaurateurApplicationInclude;
        Where: Prisma.RestaurateurApplicationWhereUniqueInput;
        Fields: "user";
        ListRelations: never;
        Relations: {
            user: {
                Shape: User;
                Types: PrismaTypes["User"];
            };
        };
    };
}
```

is this a bug in pothos generator or is it because of the use in a monorepo.
if you need a sample project or other information i would be happy to provide them.

**Comment by hayes** [comment]
You need to upgrade typescript, there was a breaking change in the ts compiler API a while back that caused this

**Comment by JipSterk** [comment]
thanks! this resolves my issue

**Comment by homoky** [comment]
Thanks for explaining the issue. But it would be great to somehow inform user about it (that he uses old typescript version).

**Comment by hayes** [comment]
@homoky adding a check in this PR https://github.com/hayes/pothos/pull/355. probably should have split it out, but I was doing multiple things at once so I guess it will just land together 🤷
